aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -260,21 +284,41 @@ spec aptos_framework::account {
         aborts_if !exists<Account>(addr);
 
         /// [high-level-req-6.1]
-        include from_scheme == ED25519_SCHEME ==> ed25519::NewUnvalidatedPublicKeyFromBytesAbortsIf { bytes: from_public_key_bytes };
-        aborts_if from_scheme == ED25519_SCHEME && ({
-            let expected_auth_key = ed25519::spec_public_key_bytes_to_authentication_key(from_public_key_bytes);
-            account_resource.authentication_key != expected_auth_key
-        });
-        include from_scheme == MULTI_ED25519_SCHEME ==> multi_ed25519::NewUnvalidatedPublicKeyFromBytesAbortsIf { bytes: from_public_key_bytes };
-        aborts_if from_scheme == MULTI_ED25519_SCHEME && ({
-            let from_auth_key = multi_ed25519::spec_public_key_bytes_to_authentication_key(from_public_key_bytes);
-            account_resource.authentication_key != from_auth_key
-        });
+        include from_scheme == ED25519_SCHEME ==>
+            ed25519::NewUnvalidatedPublicKeyFromBytesAbortsIf {
+                bytes: from_public_key_bytes
+            };
+        aborts_if from_scheme == ED25519_SCHEME
+            && (
+                {
+                    let expected_auth_key =
+                        ed25519::spec_public_key_bytes_to_authentication_key(
+                            from_public_key_bytes
+                        );
+                    account_resource.authentication_key != expected_auth_key
+                }
+            );
+        include from_scheme == MULTI_ED25519_SCHEME ==>
+            multi_ed25519::NewUnvalidatedPublicKeyFromBytesAbortsIf {
+                bytes: from_public_key_bytes
+            };
+        aborts_if from_scheme == MULTI_ED25519_SCHEME
+            && (
+                {
+                    let from_auth_key =
+                        multi_ed25519::spec_public_key_bytes_to_authentication_key(
+                            from_public_key_bytes
+                        );
+                    account_resource.authentication_key != from_auth_key
+                }
+            );
```

Comment:
This is a lot of indentation, maybe 2 space indentation would have helped here

Reply:
This is a great idea, because in our experience, deeply nested code is indeed very common in the spec syntax. If there is deeply nested code, it can adaptively adjust the number of spaces for indentation, which is a very good idea. Thank you so much.

Reply:
We will discuss more on whether to set default indentation to 2 and get back to you @robinlzw